app/assets/javascripts/admin/components/simple-list.js

```diff
@@ -0,0 +1,81 @@
+import { empty } from "@ember/object/computed";
+import Component from "@ember/component";
+import { action } from "@ember/object";
+import { on } from "discourse-common/utils/decorators";
+
+export default Component.extend({
+  classNameBindings: [":simple-list", ":value-list"],
+  inputEmpty: empty("newValue"),
+  inputDelimiter: null,
+  inputType: null,
+  newValue: "",
+  collection: null,
+  values: null,
+
+  @on("didReceiveAttrs")
+  _setupCollection() {
+    if (this.inputType === "array") {
+      this.set("collection", this.values || []);
+      return;
+    }
+
+    this.set(
+      "collection",
+      this._splitValues(this.values, this.inputDelimiter || "\n")
+    );
+  },
+
+  keyDown(event) {
+    if (event.keyCode === 13) this.addValue(this.newValue);
```

Comment:
AFAIK keyCode is getting deprecated, I know we use it the app, but well if we are writing new code, maybe we should avoid it.

https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/keyCode